.github/workflows/build.yml

```diff
@@ -36,17 +36,6 @@ jobs:
             trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
             substituters = https://cache.iog.io https://cache.nixos.org/
 
-      - name: Restore store
-        id: cache-store-restore
-        uses: actions/cache/restore@v3
-        with:
-          path: |
-              store
-              derivations
-          key: cache-store-${{ runner.os }}-${{github.run_id}}
-          restore-keys: |
-            cache-store-${{ runner.os }}
-
       - name: Import cached nix store
```

Comment:
Shouldn't we also remove this?

Reply:
I presumed this is what actually utilises the IOG cache, but I might be wrong?

Reply:
Maybe I misunderstood the intention.
Do we want to get rid of the mechanism that : exports the store to a local file, uploads it to github cache, so that it another job it can be downloaded and imported   ?

Reply:
I don't know enough of how this whole thing works, so I cannot really answer.

My question with this PR is: 
> Is there any point in having these cache-save/-restore steps in the CI build?

Or another way to put it:
> If we completely get rid of the cache-export/import mechanism for Github, do we lose anything or do we simply get faster build times?

Reply:
The whole import/export was done just for faster build times.  If we get that without it, then we can safely remove it .